dashboard/test/serializers/api/v1/pd/enrollment_flat_attendance_serializer_test.rb

```diff
@@ -24,7 +24,7 @@ class Api::V1::Pd::EnrollmentFlatAttendanceSerializerTest < ::ActionController::
       cdo_scholarship: 'Yes',
       other_scholarship: ''
     }
-    assert_equal expected, serialized
+    assert_equal expected, serialized.slice(*expected.keys)
```

Comment:
Why not just add the new fields to the expected value?

Reply:
There are 2 tests at the end to test the new extractions. Adding new fields here are unnecessary because they will just repeat the same thing multiple times.

Reply:
Ah, duh. Seems cleaner to keep it in one test but do you separate because the factories for the other tests don't include some information that you need for this test (eg, grades teaching)?